AdrianBarros/assignments/cyclicSort/lc287/lc287.py

```diff
@@ -1,15 +1,41 @@
 # Problem Statement #
 
-# We are given an unsorted array containing ‘n+1’ numbers taken from the range 1 to ‘n’. The array has only one duplicate but it can be repeated multiple times. Find that duplicate number without using any extra space. You are, however, allowed to modify the input array.
+# We are given an unsorted array containing n+1 numbers taken from the range 1 to n. The array has only one duplicate but it can be repeated multiple times. Find that duplicate number without using any extra space. You are, however, allowed to modify the input array.
```

Comment:
Nice overview here! Don't forget to include a note about the time and space complexity of your solution. It's also helpful to write some additional test cases, especially for edge cases.